.github/workflows/component.yml

```diff
@@ -1,4 +1,4 @@
-name: Continuous Integration Tests
+name: Component Tests
 
 on:
```

Comment:
Our Travis CI has daily build configured. It should be brought over here as well. [reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#schedule)

Reply:
ah yes, updated to run at the end of every day